src/connector/with_options_source.yaml

```diff
@@ -214,6 +214,33 @@ KafkaProperties:
     field_type: HashMap<String,String>
     comments: This is generated from `private_link_targets` and `private_link_endpoint` in frontend, instead of given by users.
     required: false
+  - name: region
+    field_type: String
+    required: false
```

Comment:
how about adding a `aws` prefix for these props? I am not sure if GCP or Azure has the same stuff 😅

Reply:
This also happens to those using AwsAuthProp, including bigquery kinesis and pulsar😅
https://github.com/risingwavelabs/risingwave/pull/13513/files#diff-c38146b9d974f4299ee4a603f82bbde9fda633c8f959dbb676fcfa7276d1291eR458-R460

Reply:
Maybe open another pr to add prefix for AwsAuthProp while keeping the backward compatibility

Reply:
SGTM